install/0000_90_machine-api-operator_01_prometheusrole.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This is already defined in `0000_30_machine-api-operator_09_rbac.yaml`

https://github.com/openshift/machine-api-operator/blob/b10f7c0c52f8bd98c534ec0be7e94238b6f84cb3/install/0000_30_machine-api-operator_09_rbac.yaml#L296-L312

And the same is true for the other Prometheus operator CRs.

@vikaschoudhary16 said offline that it would be tackled in another PR so you probably to coordinate there.